blazor/message/template.md

```diff
@@ -12,7 +12,7 @@ documentation: ug
 The message supports templates that allow you to customize the content with a custom structure. The content can be a string, paragraph, or any other HTML element. The template can be added directly to the `SfMessage` tags.
```

Comment:
use the below content
allows the user to

Reply:
Replaced the content.